### PR TITLE
Badges for Tech Stack and Domain

### LINKED ISSUE #4: Doc Update [ Badges for Domain and Tech Stack ]
* **I'm submitting a ...**
  - [ ] bug report
  - [ ] feature request
  - [x] change/modification
  - [ ] design issue

* **Do you want to request a *feature* or report a *bug*?**

* **What is the current behavior?**

The Domain and Tech Stack sections of the readme is currently formatted as lists which consumes a lot of space

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

* **What is the expected behavior?**

If graphic labels or badges in form of svg embeds were used instead, it will be a lot more concise and small ( horizontally laid out )

* **What is the motivation / use case for changing the behavior?**

It will allow the focus of reader to be drawn on the important parts such as usage and technical aspects instead.

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

Some useful references 

https://github.com/alexandresanlim/Badges4-README.md-Profile

https://github.com/Naereen/badges

https://github.com/badges/shields

https://shields.io/

### PR DESCRIPTION
**Pull request by jatin00000**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the current behavior?** (You can also link to an open issue here)
The Domain and Tech Stack sections of the readme is currently formatted as lists which consumes a lot of space

* **What is the new behavior (if this is a feature change)?**
Added badges for Domain and Tech Stack sections

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:
Logo is not available for some Tech Stack like matplotlib

Explain the **motivation** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

**Code formatting**

**Closing issues**

Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,26 +75,24 @@ Collect and process real time twitter data to analyse popularity of tweets with
...
-- API and Web Server Communication
-- Network Analysis
+[![Badge](https://shields.io/badge/-Data--Mining-blue?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Data--Manipulatio--and--Feature--Engineering-red?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Big--Data--Analytics-green?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Natural--Language--Processing-yellow?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-API--and--Web--Server--Communication-orange?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Network--Analysis-pink?logo=appveyor&logoColor=violet)]()
 
 <br>
```
Comment (kaustav202) [inline README.md:86]: `</br>`

File: README.md [hunk 2]
```diff
@@ -75,26 +75,24 @@ Collect and process real time twitter data to analyse popularity of tweets with
...
-
-- Data Mining
-- Data Manipulation and Feature Engineering
-- Big Data Analytics
-- Natural Language Processing
-- API and Web Server Communication
-- Network Analysis
+[![Badge](https://shields.io/badge/-Data--Mining-blue?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Data--Manipulatio--and--Feature--Engineering-red?logo=appveyor&logoColor=violet)]()&nbsp;
+[![Badge](https://shields.io/badge/-Big--Data--Analytics-green?logo=appveyor&logoColor=violet)]()&nbsp;
```
Comment (kaustav202) [inline README.md:80]: color chosen here isn't appropriate.. would be better with lighter colors

File: README.md [hunk 3]
```diff
@@ -75,26 +75,24 @@ Collect and process real time twitter data to analyse popularity of tweets with
...
+[![Badge](https://shields.io/badge/-Natural--Language--Processing-yellow?logo=appveyor&logoColor=violet)]()&nbsp;
```
Comment (kaustav202) [inline README.md:81]: same here.. just change it to a more soothing color

**Review by kaustav202 — COMMENTED** [review]
Everything else looks fine.. just make the changes mentioned. Also the logo colors should be something dark.. navy blue prob

File: README.md (hunk 1 above)
Comment (jatin00000) [reply README.md:86]: Done

File: README.md (hunk 2 above)
Comment (jatin00000) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (jatin00000) [reply README.md]: Done

**Review by kaustav202 — APPROVED** [review]
Good Work

**Comment by jatin00000** [comment]
My points are not allocated in the leaderboard. Please can you check once.

**Comment by kaustav202** [comment]
> My points are not allocated in the leaderboard. Please can you check once.

okay ill see what's the issue

**Comment by kaustav202** [comment]
@jatin00000  did you receive the points now?

**Comment by jatin00000** [comment]
No. There is typo in label. It should be like
image
It will work then.

**Pull request merged**